openbsd-games.db

```diff
@@ -1346,7 +1346,7 @@ Cover	Democracy_3_-_cover.png
 Engine	Irrlicht Engine
 Setup
 Runtime	HumblePlay
-Store	https://www.humblebundle.com/store/democracy-3
+Store	https://www.humblebundle.com/store/democracy-3 https://store.steampowered.com/app/245470 https://www.gog.com/game/democracy_3
 Hints
```

Comment:
Same as AAaaa... for the Awesome - needs HumblePlay to be useful for OpenBSD'ers.